test/python/transpiler/test_star_prerouting.py

```diff
@@ -258,8 +260,11 @@ def test_100q_grid_full_path(self, opt_level):
         )
         pm.init += StarPreRouting()
         result = pm.run(qc)
-        counts_before = AerSimulator().run(qc).result().get_counts()
-        counts_after = AerSimulator().run(result).result().get_counts()
+        with warnings.catch_warnings():
+            # TODO remove this catch once Aer stops using qiskit.qobj.common.QobjDictField`
+            warnings.filterwarnings("ignore", category=DeprecationWarning, module="qiskit")
+            counts_before = AerSimulator().run(qc).result().get_counts()
+            counts_after = AerSimulator().run(result).result().get_counts()
```

Comment:
also caught by base filter?

```suggestion
        counts_before = AerSimulator().run(qc).result().get_counts()
        counts_after = AerSimulator().run(result).result().get_counts()
```